### PR TITLE
Update icons

### PR DESCRIPTION
**Pull request by philschanely**
## Description

Adds 26 new icons and refactors the page for easier maintenance moving forward.

### CODE REVIEW

**Review by pixelflips — APPROVED** [review]
👍  - LGTM!

**Review by joe-dakroub — APPROVED** [review]
👍 Looking good. Nice cleanup!

File: app/views/pages/icon.html.erb [hunk 1]
```diff
@@ -7,517 +122,21 @@
 </p>
 <p>
   Note as well for accessibility that if the icon should have an
-  <code>aria-label</code> attribute that describes the icon's
+  <code>aria-label</code> attribute that describes the icon"s
```
Comment (pixelflips) [inline app/views/pages/icon.html.erb]: Non-blocking, but curious about the double quote replacing the apostrophe? Possible find/replace?

Comment (philschanely) [reply app/views/pages/icon.html.erb]: oops! Fixing...

**Pull request merged**